python/py-dqsegdb2/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-dqsegdb2
+version             1.0.0
+
+categories-append   science
+maintainers         {ligo.org:duncan.macleod @duncanmmacleod} openmaintainer
+
+platforms           darwin
+license             GPL-3
+
+description         Simplified python interface to DQSEGDB
+long_description    dqsegdb2 is a simplified Python implementation of the \
+                    DQSEGDB API as defined in LIGO-T1300625. \
+                    This package only provides a query interface for GET \
+                    bequests, any users wishing to make POST requests \
+                    should refer to the official 'dqsegdb' port.
+homepage            https://dqsegdb2.readthedocs.io
+
+master_sites        pypi:d/dqsegdb2
+distname            dqsegdb2-${version}
+checksums           rmd160 dc1275e619fb7bc9713903bde44a9a3f557fe936 \
+                    sha256 828a1afd0af31b4fda2bba3427836aa5cf7fb751b636140ba6e8c48d1e6aac34 \
+                    size   17980
+
+python.versions     27 36 37
+python.default_version 36
```

Comment:
I think you need to add a space before "36" to correctly conform to the modeline.